fair_mango/dataset/dataset.py

```diff
@@ -127,6 +145,19 @@ def plot_groups(
         figsize: tuple[int, int] = (16, 6),
         dpi: int = 200,
     ):
+        """Plot the distribution of the demographic groups found within the
```

Comment:
question: does this method only support "demographic" groups ? Or can it be any sensitive information ? 

thought: I agree fairness is mostly a human related topic, but it is a more broader subject and it would be reducting to only think of applying it to humans. 

Examples: 

- **Air Quality Monitoring** : If sensors are predominantly placed in urban areas, rural areas might not receive accurate air quality assessments. This could result in unfair distribution of resources and attention, leaving rural areas unprotected and uninformed about pollution risks.

- **Water Distribution in Agriculture** : Some regions or types of crops might receive less water, not because of actual need but because of historical biases or outdated assumptions. This can lead to unfair advantages and disadvantages among farmers.

Reply:
For the moment it is dedicated for the demographic groups only. It is removed in another refactoring [branch](https://github.com/datategy/Fair-Mango/tree/refactor/reduce-complexity) (still did not create a PR for it). It will be introduced again along with a more capable and complete visualisation module in the future.